packages/framer-motion/src/projection/node/create-projection-node.ts

```diff
@@ -1390,32 +1407,35 @@ export function createProjectionNode<I>({
              * layout reprojection or the final bounding box.
              */
             calcBoxDelta(
-                this.projectionDelta,
+                this.projectionDelta!,
```

Comment:
Moving object creation to `createProjectionDeltas()` removed the implicit understanding this object has now been created